zulipterminal/config/keys.py

```diff
@@ -339,6 +343,14 @@ def keys_for_command(command: str) -> List[str]:
         raise InvalidCommand(command)
 
 
+def prime_key_for_command(command: str) -> str:
+    """
+    Returns the primary key for a command,
+    currently defined as the first key
+    """
```

Comment:
Is this the `prime` or `primary` key? :) Primary may be clearer?

If the docstring then repeats the function description, we can trim it to avoid duplication.
For a user of this function 'first key' has no meaning?
A good alternative might be to say again what it means, and/or for what application? (what is 'primary key', why might it be useful?)